manifests/test-template.yml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kubevirt.io/v1alpha1
+kind: Template
+metadata:
+  name: working
+  annotations:
+    description: "OCP kubevirt linux, template"
+    tags: "kubevirt,ocp,template,linux"
+  labels:
+    miq.github.io/kubevirt-is-vm-template: true
+    miq.github.io/kubevirt-os: rhel-7
+spec:
+  template:
```

Comment:
once presets are supported: here should be a metadata section with the label `flavor: large` or `guest: Win2k (whatever)`

Reply:
OK